### PR TITLE
[frogend] Emoji categories

### PR DESCRIPTION
**Pull request by f0x52**
drop-down and freeform selection of categories with a new emoji upload
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (tsmethurst) [inline package.json:1]: is this supposed to be here?

File: web/source/settings/components/combo-box.jsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+	Combobox,
+	ComboboxItem,
+	ComboboxPopover,
+} = require("ariakit/combobox");
+
+module.exports = function ComboBox({state, items}) {
+	return (
+		<div className="form-field combobox-wrapper">
+			<label>
+				Category
```
Comment (tsmethurst) [inline web/source/settings/components/combo-box.jsx:33]: i guess in future we can make the label into a prop for this right?

File: web/source/settings/components/combo-box.jsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+				<Combobox
+					state={state}
+					placeholder="e.g., reactions"
```
Comment (tsmethurst) [inline web/source/settings/components/combo-box.jsx:36]: and the placeholder :D

File: web/source/settings/components/form/combobox.jsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+"use strict";
+
+const { useComboboxState } = require("ariakit/combobox");
```
Comment (tsmethurst) [inline web/source/settings/components/form/combobox.jsx:21]: oh this is very cool :)

**Review by tsmethurst — COMMENTED** [review]
Looks good to me :) few small comments

File: package.json (hunk 1 above)
Comment (f0x52) [reply package.json]: nope oops

**Pull request merged**